frontend-new/src/pages/index.vue

```diff
@@ -1,12 +1,187 @@
+    {id: 'mit', label: 'MIT'},
+    {id: 'apache', label: 'APACHE'},
+];
 </script>
 
 <template>
-  <div>Index Page {{ t("general.close") }}</div>
+    <div class="flex flex-col items-center pt-10">
+        <h2 class="text-3xl font-bold uppercase">Find your favorite plugins</h2>
+        <!-- Search Bar & Sorting button -->
+        <div class="flex flex-row mt-6 rounded-md big-box-shadow">
+            <!-- Search Bar -->
+            <input
+                class="rounded-l-md p-3 w-[80vw] max-w-800px focus-visible:(border-white) text-black" type="text"
+                placeholder="Search in 1 projects, proudly made by the community...">
+            <!-- Sorting Button -->
+            <div class="rounded-r-md w-100px bg-gradient-to-r from-[#004ee9] to-[#367aff]">
+                <Menu>
+                    <MenuButton
+                        class="rounded-r-md h-1/1 text-left font-semibold flex flex-row items-center gap-2 text-white p-2">
+                        <span>Sort by</span>
+                        <icon-mdi-sort-variant class="text-[1.2em] pointer-events-none overflow-hidden"/>
+                    </MenuButton>
+                    <transition
+                        enter-active-class="transition duration-100 ease-out"
+                        enter-from-class="transform scale-95 opacity-0"
+                        enter-to-class="transform scale-100 opacity-100"
+                        leave-active-class="transition duration-75 ease-out"
+                        leave-from-class="transform scale-100 opacity-100"
+                        leave-to-class="transform scale-95 opacity-0"
+                    >
+                        <MenuItems
+                            class="absolute flex flex-col z-10 background-header shadow1 rounded-md border-top-primary">
+                            <MenuItem
+                                v-for='sorter in sorters'
+                                :key='sorter.id'
+                                v-slot="{ active }">
+                                <a
+                                    :class='{ "bg-gradient-to-r from-[#004ee9] to-[#367aff] text-white": active }'
+                                    class="p-2">
+                                    {{ sorter.label }}
+                                </a>
+                            </MenuItem>
+                        </MenuItems>
+                    </transition>
+                </Menu>
+            </div>
+        </div>
+
+    </div>
+    <div
+        class="projectsAndSidebar p-4 mt-5 w-screen max-w-1200px flex justify-around m-auto flex-col md:flex-row gap-y-6 md:gap-y-0 md:gap-x-6">
+        <!-- Projects -->
+        <div class="md:(w-2/3 min-w-2/3 max-w-2/3) min-h-800px bg-gray-200 rounded-md">
+
+        </div>
+        <!-- Sidebar -->
+        <div
+            class="flex flex-col gap-4 bg-gradient-to-r from-[#004ee9] to-[#367aff] rounded-md min-w-300px min-h-800px p-4 text-white">
```

Comment:
this min width doesn't check out, at 800px screen width it forces the sidebar to escape the container.